src/stream.ts

```diff
@@ -85,8 +89,41 @@ export class LocalStream extends MediaStream {
     return stream.getTracks()[0];
   }
 
+  addTrack(track: MediaStreamTrack) {
+    super.addTrack(track);
+
+    if (this.pc) {
+      if (track.kind === 'video' && this.options.simulcast) {
+        this.pc.addTransceiver(track, {
+          streams: [this],
+          direction: 'sendrecv',
+          sendEncodings: this.options.encodings
+            ? this.options.encodings
+            : [
+                {
+                  rid: 'f',
+                },
+                {
+                  rid: 'h',
+                  scaleResolutionDownBy: 2.0,
+                  maxBitrate: 150000,
```

Comment:
we should make the maxbitrate configurable with defaults.

Reply:
it is possible to define them by passing encodings on the `Constraints` object

```
export interface Constraints extends MediaStreamConstraints {
  resolution: string;
  codec: string;
  simulcast: boolean;
  encodings?: RTCRtpEncodingParameters[];
}
```

Reply:
Yes, but Im not sure if we should expose encodings, since rids are hardcoded in sever. also encodings orders matters.

Reply:
ok. let me rethink the api then